test/UnitTest/data/apps/sfvt/test-read-app-no-permission/config/authorization/policy.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xacml:Policy xmlns:xsl="http://www.w3.org/2001/XMLSchema-instance" xmlns:xacml="urn:oasis:names:tc:xacml:3.0:core:schema:wd-17" PolicyId="urn:altinn:example:policyid:1" Version="1.0" RuleCombiningAlgId="urn:oasis:names:tc:xacml:3.0:rule-combining-algorithm:deny-overrides">
+    <xacml:Target/>
+    <xacml:Rule RuleId="urn:altinn:example:ruleid:1" Effect="Permit">
+        <xacml:Description>A rule giving sfvt the right to read, write, complete and instantiate a given app of sfvt/dgm-signering</xacml:Description>
+        <xacml:Target>
+            <xacml:AnyOf>
+                <xacml:AllOf>
+                    <xacml:Match MatchId="urn:oasis:names:tc:xacml:1.0:function:string-equal">
+                        <xacml:AttributeValue DataType="http://www.w3.org/2001/XMLSchema#string">sfvt</xacml:AttributeValue>
+                        <xacml:AttributeDesignator AttributeId="urn:altinn:org" Category="urn:oasis:names:tc:xacml:3.0:attribute-category:resource" DataType="http://www.w3.org/2001/XMLSchema#string" MustBePresent="false" />
+                    </xacml:Match>
+                    <xacml:Match MatchId="urn:oasis:names:tc:xacml:1.0:function:string-equal">
+                        <xacml:AttributeValue DataType="http://www.w3.org/2001/XMLSchema#string">test-read-app</xacml:AttributeValue>
+                        <xacml:AttributeDesignator AttributeId="urn:altinn:app" Category="urn:oasis:names:tc:xacml:3.0:attribute-category:resource" DataType="http://www.w3.org/2001/XMLSchema#string" MustBePresent="false" />
+                    </xacml:Match>
+                </xacml:AllOf>
+            </xacml:AnyOf>
```

Comment:
This is the resource category. We also need a section for access-subject. Look at one of the existing policy files.

Missing section for Category `urn:oasis:names:tc:xacml:1.0:subject-category:access-subject`

Reply:
The file is in a folder with name `test-read-app-no-permission`. Resource inside the file is `test-read-app`.

The tests are using a simple mocked authorization component and it's using naming conventions to find correct policy file (I think). Make sure the resource inside the file match the rest.